### PR TITLE
Implement empty sign up screen

### PR DESCRIPTION
**Pull request by BKinya**
# Scope

_Please make sure to read https://github.com/droidconKE/droidconKE2022Android/docs/CONTRIBUTING.md
and check that you understand and have followed it as best as possible 

Changes
- Implemented an empty sign up screen.
- Implemented navigation to and from the login screen. 
- Made sign up and login screens scrollable

_ please check the below boxes
- [ x ] I have followed the coding conventions
- [ ] I have added/updated necessary tests
- [ x ] I have tested the changes added on a physical device

## Closes/Fixes Issues
_Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick
in when this is merged_

## Other testing QA Notes
_What have you tested specifically and what possible impacts/areas there are that may need retesting
by others._

Please add a screenshot (if necessary)
Screenshot_20220723-075748_DroidconKE2022
Screenshot_20220723-080016_DroidconKE2022

### CODE REVIEW

File: presentation/src/main/java/com/android254/presentation/auth/view/LoginScreen.kt [hunk 1]
```diff
@@ -45,7 +43,11 @@ import com.android254.presentation.common.theme.DroidconKE2022Theme
 import com.android254.presentation.common.theme.Montserrat
 
 @Composable
-fun LoginScreen(darkTheme: Boolean = isSystemInDarkTheme(), navController: NavHostController) {
+fun LoginScreen(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    navController: NavHostController,
+    goToSignUp: () -> Unit = {}
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/auth/view/LoginScreen.kt:49]: `goToSignUp` -> `navigateToSignUp`

File: presentation/src/main/java/com/android254/presentation/auth/view/SignUpScreen.kt [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android254.presentation.R
+import com.android254.presentation.common.components.DroidConTextField
+import com.android254.presentation.common.components.SocialAuthButton
+import com.android254.presentation.common.theme.DroidconKE2022Theme
+import com.android254.presentation.common.theme.Montserrat
+
+@Composable
+fun SignUpScreen(goToLogin: () -> Unit = {}, isDarkTheme: Boolean = isSystemInDarkTheme()) {
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/auth/view/SignUpScreen.kt:44]: `goToLogin` -> `navigateToLogin`

File: presentation/src/main/java/com/android254/presentation/auth/view/SignUpScreen.kt [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                        painterResource(id = R.drawable.topbar_bg_sign_up)
+                    },
+                    contentDescription = stringResource(id = R.string.login_screen_bg_image_description),
+                    contentScale = ContentScale.FillBounds
+                )
+                LargeTopAppBar(
+                    title = { Text(stringResource(R.string.sign_up_label)) },
+                    navigationIcon = {
+                        IconButton(
+                            onClick = { /* doSomething() */ },
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/auth/view/SignUpScreen.kt:62]: `onClick = { /* doSomething() */ },` -> `onClick = { Unit },`

**Review by chepsi — APPROVED** [review]
Minor improvements. LGTM

**Review by wangerekaharun — APPROVED** [review]
Looks good, address @chepsi comments and we'll proceed to merge this

**Comment by BKinya** [comment]
I've made the changes. 😀

**Comment by misshannah** [comment]
@BKinya, tiny question.. shouldn't the text be **"Already have an account?"** instead of  "Already have an account!"

**Comment by BKinya** [comment]
> @BKinya, tiny question.. shouldn't the text be **"Already have an account?"** instead of "Already have an account!"

Hi @misshannah. I've corrected the typo 🙂.

**Pull request merged**